cryptoapis/src/adapter.ts

```diff
@@ -23,20 +23,12 @@ export const execute: ExecuteWithConfig<Config> = async (request, config) => {
       response = await price.execute(config, request)
       break
     }
-    case 'difficulty':
-    case 'height': {
-      response = await bc_info.execute(config, request)
-      break
-    }
     case balance.Name: {
       return balance.makeExecute(config)(request)
     }
     default: {
```

Comment:
I understand your concern. If we don't do this though, it would be hard to pass arbitrary "endpoint" parameters to retrieve blockchain info(if we want to maintain backwards compatibility for this certain call). We might need to create a map that declares all or almost most of them. Or we can just support only these 2 declared/documented.

Reply:
Changing these on adapter.ts duplication PR